src/main.rs

```diff
@@ -25,6 +25,11 @@ struct Args {
     #[clap(short, long)]
     deref: bool,
 
+    /// Edit the file in-place
+    #[clap(short, long)]
+    #[clap(requires("val"))]
```

Comment:
```suggestion
    #[clap(requires("write"))]
```
So apparently since arg group "write" exists changing this makes it so any writing action is compatible with `--inplace`, but anything else like reading a value won't work.

Also can you squash after making the changes?